handlers/handlers.go

```diff
@@ -11,6 +11,7 @@ import (
 	"runtime/debug"
 	"strings"
 
+	"github.com/Masterminds/sprig/v3"
```

Comment:
Didn't know this was a thing! Looks pretty neat, might I ask where it is being used though 😅

Reply:
Was used in `get-livestream.tmpl`, except it isn't anymore since I moved it all to JS 😅 so can probably be removed actually

Reply:
Nice lol